src/flat-file-adapter.js

```diff
@@ -0,0 +1,26 @@
+const flatfile = require('flat-file-db')
+const promise = require('promise')
+
+const db = flatfile.sync(process.env.DB_NAME || 'views.db')
+
+
+module.exports = {
+  put: promise.denodeify(db.put.bind(db)),
+
+  has: (key) => Promise.resolve(db.has(key)),
+  get: (key) => Promise.resolve(db.get(key)),
+  keys: () => Promise.resolve(db.keys()),
+
+  getAll: async function getAll(options) {
+    const data = {}
+    const keys = await module.exports.keys()
+
+    keys
+      .filter(key => String(options.filter) === 'false' ? true : key.startsWith(options.pathname))
```

Comment:
The `filter` option is not officially documented I don't think, so let's get deprecate it. (I don't think anybody uses that) That's just more unnecessary burden on the adapters, it should be easy to write one so lots of people do!